mlir/test/Dialect/Vector/canonicalize.mlir

```diff
@@ -3375,6 +3375,42 @@ func.func @negative_from_elements_to_constant() -> vector<1x!llvm.ptr> {
   return %b : vector<1x!llvm.ptr>
 }
 
+// -----
+
+// CHECK-LABEL: @negative_from_elements_poison
+//       CHECK:   %[[VAL:.*]] = ub.poison : vector<2xf32>
+//       CHECK:   return %[[VAL]] : vector<2xf32>
+func.func @negative_from_elements_poison_f32() -> vector<2xf32> {
```

Comment:
this isn't "negative", right? The canonicalization is happening?

Reply:
Let me know if action is required from me.

Reply:
@jhalakpatel , you missed my request to document these tests:
> I suggest adding a comment to clarify.

;-) In the spirit of maintaining healthy test hygiene: https://github.com/llvm/llvm-project/pull/160106. Could you review and approve if it makes sense? Thanks!